robots/panda_description/urdf/panda.urdf

```diff
@@ -225,56 +231,121 @@
                 <mesh filename="package://example-robot-data/robots/panda_description/meshes/collision/hand.stl" />
             </geometry>
         </collision>
-    </link>
-    <link name="panda_leftfinger">
         <inertial>
+            <origin rpy="0 0 0" xyz="-0.01 0 0.03"/>
+            <mass value="0.73"/>
+            <inertia ixx="0.001" ixy="0" ixz="0" iyy="0.0025" iyz="0" izz="0.0017"/>
+        </inertial>
+    </link>
+    <!-- Define the hand_tcp frame -->
+    <link name="panda_hand_tcp">
+	<inertial>
           <mass value="0"/>
           <origin rpy="0 0 0" xyz="0 0 0"/>
           <inertia ixx="0" ixy="0" ixz="0" iyy="0" iyz="0" izz="0"/>
         </inertial>
+    </link>
+    <joint name="panda_hand_tcp_joint" type="fixed">
+        <origin rpy="0 0 0" xyz="0 0 0.1034"/>
+        <parent link="panda_hand"/>
+        <child link="panda_hand_tcp"/>
+    </joint>
+    <link name="panda_leftfinger">
         <visual>
             <geometry>
                 <mesh filename="package://example-robot-data/robots/panda_description/meshes/visual/finger.dae" />
             </geometry>
         </visual>
+        <!-- screw mount -->
         <collision>
+            <origin rpy="0 0 0" xyz="0 18.5e-3 11e-3"/>
             <geometry>
-                <mesh filename="package://example-robot-data/robots/panda_description/meshes/collision/finger.stl" />
```

Comment:
Is this intended to be? If so, why?

Reply:
The upstream Gazebo simulation URDF now uses a box. I believe the finger was a mesh that was approximately a box previously, so no it's cheaper (and numerically more stable for simulation) to use a primitive shape